nodewatcher/modules/monitor/http/survey/processors.py

```diff
@@ -0,0 +1,80 @@
+from django.utils.translation import gettext_noop as _
+
+from django_datastream import datastream
+
+from nodewatcher.core.monitor import processors as monitor_processors
+from nodewatcher.modules.monitor.sources.http import processors as http_processors
+from nodewatcher.modules.monitor.datastream import fields as ds_fields, models as ds_models
+from nodewatcher.modules.monitor.datastream.pool import pool as ds_pool
+
+
+class SurveyInfoStreams(ds_models.RegistryRootStreams):
+    neighbor_graph = ds_fields.GraphField(tags={
+        'title': _("Neighbor topology"),
+        'description': _("Neighbor topology."),
+    })
+
+    def get_module_name(self):
+        return 'monitor.http.survey'
+
+    def get_stream_highest_granularity(self):
+        return datastream.Granularity.Hours
```

Comment:
Yes, but your comment "Ah, you are right" was confusing because you expressed agreement with Kostko who was doubting whether this was the correct granularity?

Anyway, I didn't change it and added comments about computing it.